### PR TITLE
[ready] KPVD, Theodore Francis State Memorial Airport

### PR DESCRIPTION
**Pull request by eliuuk**
My first US airport. I don't particularly like doing US airports, as they require some scavenging (usually), but this one has been quite simple so far!

With 2 STARs and no SIDs, it's quite similar to EGNM, but with less traffic.

Requested by @Casper52250 in #670.

Thank you, @erikquinn for the videomap!

Reopened due to an issue when rebasing - apparently the old branch didn't have anything in common

### CODE REVIEW

File: assets/scripts/bundle.js [hunk 1]
```diff
@@ -42244,6 +42244,10 @@ var AIRPORT_LOAD_LIST = exports.AIRPORT_LOAD_LIST = [{
     level: 'medium',
     name: 'San Francisco International Airport &#9983'
 }, {
+    icao: 'kpvd',
+    level: 'beginner',
+    name: 'Theodore Francis State Memorial Airport'
+}, {
```
Comment (n8rzz) [inline assets/scripts/bundle.js:42250]: be sure to also add these lines to `https://github.com/zlsa/atc/blob/gh-pages/src/assets/scripts/airport/airportLoadList.js`

**Comment by eliuuk** [comment]
I'll sort the conflicts - standby!

**Comment by eliuuk** [comment]
Conflicts resolved!

**Comment by eliuuk** [comment]
Relocated to openscope/openscope#101, closing!

**Pull request closed without merging**